### PR TITLE
Misc HASH app styling cleanup

### PR DESCRIPTION
**Pull request by MaggieAppleton**
## 🌟 What is the purpose of this PR?

Cleans up some spacing and styling issues on the alpha.hash.ai instance - specifically in the sidebar and header. Just a few small things to help us get the app ready for alpha users.

## 🔍 What does this change?

- Adjusts margins and padding of all the elements in the sidebar: workspace switcher, section headers, and individual page and type items
- Aligns all elements in the sidebar
- Hides the "more options" ellipsis on individual page items in the sidebar unless a user is hovering over them
- Lightens icon colour on pages (temp fix until we can switch to a new icon library with a lighter line weight)
- Tightens up spacing in the header and fixes alignment
- Add an "alpha" label next to the HASH logo

## ❓ How to test this?

Run this branch locally, create a page, create a type, then click around the sidebar to see how hovers and spacing behave.

## 👉 [Video Walkthrough](https://share.cleanshot.com/8ttF7A)

CleanShot 2022-06-17 at 14 33 03@2x

### CODE REVIEW

File: packages/hash/frontend/src/shared/layout/layout-with-sidebar/account-entity-type-list/entity-type-item.tsx [hunk 1]
```diff
@@ -96,7 +98,7 @@ export const EntityTypeItem: VFC<EntityTypeItemProps> = ({
           size="medium"
           unpadded
           sx={({ palette }) => ({
-            color: palette.gray[40],
+            color: "transparent",
```
Comment (teenoh) [inline packages/hash/frontend/src/shared/layout/layout-with-sidebar/account-entity-type-list/entity-type-item.tsx:101]: We should add the color back in `:focus-visible`, so it doesn't look like this when tabbed into

Screenshot 2022-06-17 at 16 03 32

File: packages/hash/frontend/src/shared/layout/layout-with-sidebar/account-page-list/page-tree-item.tsx [hunk 2]
```diff
@@ -141,10 +141,10 @@ const CustomContent = React.forwardRef((props: TreeItemContentProps, ref) => {
           unpadded
           className="page-menu-trigger"
           sx={({ palette }) => ({
-            color: palette.gray[40],
+            color: "transparent",
```
Comment (teenoh) [inline packages/hash/frontend/src/shared/layout/layout-with-sidebar/account-page-list/page-tree-item.tsx:144]: See https://github.com/hashintel/hash/pull/666/files?diff=unified&w=1#r900212054

File: packages/hash/frontend/src/shared/layout/layout-with-sidebar/sidebar.tsx [hunk 3]
```diff
@@ -80,7 +81,7 @@ export const PageSidebar: VoidFunctionComponent = () => {
         href="/"
         tooltipTitle="Pages you’ve recently visited"
       /> */}
-      <Box sx={{ mb: 1.5 }} />
+      {/* <Box sx={{ mb: 1 }} /> */}
```
Comment (teenoh) [inline packages/hash/frontend/src/shared/layout/layout-with-sidebar/sidebar.tsx:84]:
```suggestion
      {/* <Box sx={{ mb: 1 }} /> */}
```
We can go ahead deleting this

**Review by teenoh — CHANGES_REQUESTED** [review]
Thanks @MaggieAppleton. Tested and it looks nicer! Noticed an issue where the ellipsis menu isn't visible when tabbed into. Other than that, this looks good to go

File: packages/hash/frontend/src/shared/layout/layout-with-sidebar/account-entity-type-list/entity-type-item.tsx (hunk 1 above)
Comment (MaggieAppleton) [reply packages/hash/frontend/src/shared/layout/layout-with-sidebar/account-entity-type-list/entity-type-item.tsx]: Good catch - all fixed now

File: packages/hash/frontend/src/shared/layout/layout-with-sidebar/sidebar.tsx (hunk 3 above)
Comment (MaggieAppleton) [reply packages/hash/frontend/src/shared/layout/layout-with-sidebar/sidebar.tsx]: 👍 All done

**Comment by nonparibus** [comment]
Teenoh OOO so tagging Ciaran in.

**Review by CiaranMn — APPROVED** [review]

**Pull request merged**